### PR TITLE
Update Units.json

### PR DESCRIPTION
**Pull request by Thienanbad**
Fixing spelling

### CODE REVIEW

File: jsons/Units.json [hunk 1]
```diff
@@ -1196,7 +1196,7 @@
 		"obsoleteTech": "Military Science",
 		"upgradesTo": "Cavalry",
 		"uniques": ["Can move after attacking","No defensive terrain bonus",
-			"[-33]% Strength <vs cities>","Consumes [1] [Horses]","1 additional attack per turn"],
+			"[-33]% Strength <vs cities>","Consumes [1] [Horses]","1 additional attacks per turn"],
```
Comment (yairm210) [inline jsons/Units.json:1199]:
```suggestion
			"[-33]% Strength <vs cities>","Consumes [1] [Horses]","[+1] additional attacks per turn"],
```

Comment (yairm210) [reply jsons/Units.json:1199]: See https://yairm210.github.io/Unciv/Modders/uniques/

**Comment by Thienanbad** [comment]
I don't see any different

Vào 4:01, Th 3, 13 thg 9, 2022 Yair Morgenstern ***@***.***>
đã viết:

**Comment by yairm210** [comment]
Number needs to be wrapped in [square brackets]

**Pull request merged**